### PR TITLE
Create SES::sendEmail

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by Nyholm** [comment]
Hm, those missing classes was not generated..

File: src/Ses/Input/Body.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     * An object that represents the version of the message that is displayed in email clients that support HTML. HTML
+     * messages can include formatted text, hyperlinks, images, and more.
+     *
+     * @var Content|null
+     */
+    private $Html;
+
+    /**
+     * @param array{
+     *   Text?: \AsyncAws\Ses\Input\Text|array,
```
Comment (jderusse) [inline src/Ses/Input/Body.php]: The docblock on property says it's `Content`

Comment (Nyholm) [reply src/Ses/Input/Body.php]: Yeah. But the class was not generated.

Comment (jderusse) [reply src/Ses/Input/Body.php]: maybe the class does not exist and the issue is the doc lock here ?

Comment (Nyholm) [reply src/Ses/Input/Body.php]: Yeah, the doc block is wrong.

**Comment by Nyholm** [comment]
Wow, you are quick. Thank you

**Pull request merged**